### PR TITLE
CU-86a129e36 - Login Screen Implementation

### PR DESCRIPTION
**Pull request by yumiuehara**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a129e36 Login Screen Implementation](https://app.clickup.com/t/86a129e36)

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a129e36 Login Screen Implementation](https://app.clickup.com/t/86a129e36)

File: src/renderer/src/pages/Login/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+type TFormData = {
+  password: string
+}
+
+export const LoginPage = () => {
+  const { t } = useTranslation('pages', { keyPrefix: 'login' })
+  const form = useForm<TFormData>()
+  const location = useLocation()
+  const wallets = useSelectorRef(state => state.wallet.data)
```
Comment (raulduartep) [inline src/renderer/src/pages/Login/index.tsx:22]:
```suggestion
import { selectWallets } from '@renderer/store/wallet/SelectorWallet'

  const wallets = useSelectorRef(selectWallets)
```

File: src/renderer/src/pages/Login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  const accounts = useSelectorRef(state => state.account.data)
```
Comment (raulduartep) [inline src/renderer/src/pages/Login/index.tsx:23]:
```suggestion
import { selectAccounts } from '@renderer/store/account/SelectorAccount'

  const accounts = useSelectorRef(selectAccounts)
```

File: src/renderer/src/pages/Login/index.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  const bsAggregator = useSelectorRef(state => state.blockchain.bsAggregator)
+  const isFirstTimeRef = useSelectorRef(state => state.settings.isFirstTime)
```
Comment (raulduartep) [inline src/renderer/src/pages/Login/index.tsx:25]: Change the useSelectorRef to  useSelector

File: src/renderer/src/pages/Login/index.tsx (hunk 1 above)
Comment (raulduartep) [inline src/renderer/src/pages/Login/index.tsx]:
```suggestion
  import { selectWallets } from '@renderer/store/wallet/SelectorWallet'
	
	  const wallets = useSelectorRef(selectWallets)
```

File: src/renderer/src/pages/Login/index.tsx (hunk 2 above)
Comment (raulduartep) [inline src/renderer/src/pages/Login/index.tsx]:
```suggestion
  import { selectAccounts } from '@renderer/store/account/SelectorAccount'
	
	  const accounts = useSelectorRef(selectAccounts)
```

**Review by raulduartep — CHANGES_REQUESTED** [review]

File: src/renderer/src/pages/Login/index.tsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  password: string
+}
+
+export const LoginPage = () => {
+  const { t } = useTranslation('pages', { keyPrefix: 'login' })
+  const form = useForm<TFormData>()
+  const location = useLocation()
+  const wallets = useSelectorRef(selectWallets)
+  const accounts = useSelectorRef(selectAccounts)
+  const bsAggregator = useSelectorRef(state => state.blockchain.bsAggregator)
```
Comment (raulduartep) [inline src/renderer/src/pages/Login/index.tsx:26]:
```suggestion
  const bsAggregator = useSelectorRef(selectBsAggregator)
```

Comment (raulduartep) [inline src/renderer/src/pages/Login/index.tsx]:
```suggestion
  const bsAggregator = useSelectorRef(selectBsAggregator)
```

**Review by raulduartep — CHANGES_REQUESTED** [review]

**Review by raulduartep — APPROVED** [review]

**Pull request merged**